phpunit/class-override-script-test.php

```diff
@@ -40,7 +40,7 @@ public function test_localizes_script() {
 		);
 
 		$script = $wp_scripts->query( 'gutenberg-dummy-script', 'registered' );
-		$this->assertEquals( array( 'dependency', 'wp-i18n' ), $script->deps );
+		$this->assertEquals( array( 'dependency' ), $script->deps );
```

Comment:
Note: in #49994, the function which injects wp-i18n was changed. According to @gziolo, this is expected:

> I fixed [too many scripts loading] with https://github.com/WordPress/gutenberg/commit/0cdab0869726d5ebde7f33357456c92fac1b1e2e. I'm not sure why that code was enforcing including wp-i18n as a dependency to all scripts coming from the Gutenberg plugin 🤷🏻
>
> I replicated the logic from WordPress core that resolves the issue and ensures that translations still work where applicable

Since the phpunit tests were not running correctly, it makes sense that the phpunit tests were not updated in that PR